bin/check-consul-leader.rb

```diff
@@ -77,7 +87,13 @@ def strip_ip(str)
   end
 
   def run
-    r = RestClient::Resource.new("#{config[:scheme]}://#{config[:server]}:#{config[:port]}/v1/status/leader", timeout: 5).get
+    options = { timeout: 5,
```

Comment:
I think we should expose the timeout as an option, someone will inevitably say that it's reasonable to wait more than 5 seconds.